### ISSUE TITLE
How the app.current_tenant is used?

### ISSUE THREAD

**Issue by ZedZipDev** [bytefish/AspNetCoreMultitenancy#2]
Hi, thank you for very interesting example.
Could you please explain how this session variable used? I did not find.

```
        private void PrepareCommand(DbCommand cmd, Tenant tenant)
        {
            cmd.CommandText = $"SET app.current_tenant = '{tenant.Name}'";
        }
```
It works great. By the way, I have added Swagger and now the app can be tested better .

**Comment by bytefish** [comment]
Sounds great! Feel free to make a PR, I’d be happy to merge it. 😊 The `app.current_tenant` session variable is used for the RLS Policy here: 

* https://github.com/bytefish/AspNetCoreMultitenancy/blob/d658494128d34774c6e66ce00805803b70be40f0/Sql/create.sql#L50

**Comment by ZedZipDev** [comment]
Oh, sorry, my mistake, I did check the c# code only :-) .
Could you please say me: how to do it better? Fork the project, then do my changes and then PR?

**Comment by bytefish** [comment]
Yes, I think it's best to fork the repository and add the Swagger Middleware to the `Startup`. 👍

**Comment by ZedZipDev** [comment]
Ok, und noch eine Frage:

Is it possible to do the same but for SQL Server database? What do you think?

**Comment by bytefish** [comment]
Yes, T-SQL has a slightly different syntax, but it's possible to provide the Tenant Identifier from a middle-tier application as well, see:

* https://learn.microsoft.com/en-us/sql/relational-databases/security/row-level-security?view=sql-server-ver16#MidTier

**Comment by ZedZipDev** [comment]
ok, please check my PR.
